packages/create-plugin/src/constants.ts

```diff
@@ -21,6 +21,7 @@ export const TEMPLATE_PATHS: Record<string, string> = {
   scenesapp: path.join(TEMPLATES_DIR, 'scenes-app'),
   backend: path.join(TEMPLATES_DIR, 'backend'),
   backendApp: path.join(TEMPLATES_DIR, 'backend-app'),
+  provisioning: path.join(TEMPLATES_DIR, 'provisioning-template'),
```

Comment:
Let's change the name of this directory to `provisioning` (as I see we are not appending "-template" to the other template sub-directories).

Also, after we remove the `README.md` and `docker-compose.yaml` from this folder, we can make it look like this:
```
/templates/
   /provisioning
     /dashboards/
     /datasources
     ...
```

Reply:
Added benefit here is the root docker-compose already mounts the `provisioning` directory so whatever gets scaffolded will be available inside the dev env. 👍